client/src/Poem.jsx

```diff
@@ -12,7 +12,9 @@ PoemLine.propTypes = {
 
 const Poem = ({ poem }) => {
   const poemLineNodes = poem.text.split(/\n/).map((line, index) => (
-    <PoemLine line={line} key={index} />
+    (line.trim().length != 0) ?
+      <PoemLine line={line} key={index} /> :
+      <PoemLine line="&nbsp;" key={index} />
```

Comment:
Consider storing the line contents in a variable rather than duplicating the PoemLine element.